kuma/search/filters.py

```diff
@@ -192,3 +194,21 @@ def filter_queryset(self, request, queryset, view):
         queryset = queryset.extra(facets=facets)
 
         return queryset
+
+
+class HighlightFilterBackend(BaseFilterBackend):
+    """
+    A django-rest-framework filter backend that adds search term highlighting.
+
+    If the query string `highlight=0` is passed, we bypass highlighting.
+
```

Comment:
Hmm, I always thought that was a pep8 thing but now that I look I can't find it.